Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js

```diff
@@ -389,6 +403,23 @@ WI.CSSManager = class CSSManager extends WI.Object
         return false;
     }
 
+    overrideUserPreference(preference, value)
+    {
+        for (let target of WI.targets) {
+            // COMPATIBILITY (macOS 13.0, iOS 16.0): `Page.overrideUserPreference()` did not exist yet.
+            if (target.hasCommand("Page.overrideUserPreference")) {
+                if (value) {
+                    target.PageAgent.overrideUserPreference(preference, value);
+                    this._overridenUserPreferences.set(preference, value);
+                }
+                else {
+                    target.PageAgent.overrideUserPreference(preference); // Pass no value to remove the override and restore the system default.
+                    this._overridenUserPreferences.delete(preference);
+                }
```

Comment:
I think this can be simplified a bit:
```
target.PageAgent.overrideUserPreference(preference, value);
if (value)
    this._overridenUserPreferences.set(preference, value);
else
    this._overridenUserPreferences.delete(preference);
```

Reply:
Good catch.